test/cluster/test_multidc.py

```diff
@@ -464,3 +465,68 @@ async def test_restart_with_prefer_local(request: pytest.FixtureRequest, manager
 
     await manager.server_stop_gracefully(s_info.server_id)
     await manager.server_start(s_info.server_id)
+
+@pytest.mark.asyncio
+async def test_vnodes_describe_ring(request: pytest.FixtureRequest, manager: ManagerClient, build_mode) -> None:
```

Comment:
Don't we already have a test for this?

We should be able to test these APIs by instantiating a topology object with arbitrary node configuration, instead of having to launch a cluster.